enterprise/api.py

```diff
@@ -7,10 +7,21 @@
 from enterprise.models import EnterpriseCustomerBrandingConfiguration
 
 
-def get_enterprise_branding_info(provider_id=None):
+def get_enterprise_branding_info_by_provider_id(provider_id=None):  # pylint: disable=invalid-name
     """
-    Return the EnterpriseCustomer branding information.
+    Return the EnterpriseCustomer branding information based on enterprise customer provider_id.
+
+    Provider_id: There is 1:1 relation b/w EnterpriseCustomer and SAML Identity provider.
```

Comment:
@asadiqbal08 behavior was updated [here](https://github.com/edx/edx-enterprise/pull/16/files#diff-ce3a6d48e7f42961c07cd219f505d179R27), now we can attach any SSO provider.